### PR TITLE
Gene matrix, fastq sample names, and subfamilies config schema 

### PR DESCRIPTION
**Pull request by jjjoelle**
Issues: #15 and #12

### CODE REVIEW

File: workflow/simulate/concat_txte_simulations.py [hunk 1]
```diff
@@ -23,13 +23,31 @@
...
+    # add a gene id column
+    genes_df = genes_gtf.df
+    genes_df.rename(columns={"transcript_id": "tx_id"}, inplace=True)
+    tx_counts = tx_counts.merge(
+        genes_df[["tx_id", "gene_id"]], on="tx_id"
+    ).drop_duplicates()
+    return tx_counts.groupby("gene_id").sum().drop(columns=["tx_id"])
+
+
+# TODO: check this
```
Comment (mikecuoco) [inline workflow/simulate/concat_txte_simulations.py:46]: @jjjoelle were you able to verify the output?

File: .test/config.yaml [hunk 2]
```diff
@@ -8,7 +8,8 @@ gencode_gtf: ngs-test-data/rnaseq/ref/txome.chr22.gtf
 # list of transcriptomes generate and simulate reads from
 txomes:
   test_txome:
-    rmsk_query: "repName == 'L1HS' & length > 6000"
+    rmsk_query:
```
Comment (mikecuoco) [inline .test/config.yaml:11]: @jjjoelle can you rename this "te_subfamilies", and also allow for it not to be specified

File: workflow/simulate/concat_txte_simulations.py (hunk 1 above)
Comment (mikecuoco) [reply workflow/simulate/concat_txte_simulations.py:46]: Why this function in `concat_txte_simulations.py` and `simulate_te_counts.py`? Is it possible to remove that redundancy?

**Review by mikecuoco — CHANGES_REQUESTED** [review]

File: workflow/simulate/concat_txte_simulations.py (hunk 1 above)
Comment (jjjoelle) [reply workflow/simulate/concat_txte_simulations.py:46]: I originally made the function in `simulate_te_counts.py` but then that file got deleted. I updated and deleted `simulate_te_counts.py`. 
 
 And the output should be correct

**Pull request merged**